VideoLocker/src/main/java/org/edx/mobile/view/NavigationFragment.java

```diff
@@ -231,6 +233,7 @@ public void onClick(View v) {
             @Override
             public void onClick(View v) {
                 environment.getRouter().forceLogout(getActivity(), environment.getSegment(), environment.getNotificationDelegate());
+                SecurityUtil.clearUserData(MainApplication.application);
```

Comment:
There was a requirement to only clear a user's data when he explicitly logs out i.e. using the Logout button. Another case of automatic logout is the token expiry case.

Javadoc of forceLogout function:
`````java
    /**
     * this method can be called either through UI [ user clicks LOGOUT button],
     * or programmatically
     */